src/NuGet.Core/NuGet.Protocol/Converters/NuGetFrameworkConverter.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Newtonsoft.Json;
+using NuGet.Frameworks;
+
+namespace NuGet.Protocol.Converters
+{
+    internal class NuGetFrameworkConverter : JsonConverter
```

Comment:
I think we can use generic version of the JsonConverter for better type safety and may be performance.

https://www.newtonsoft.com/json/help/html/CustomJsonConverterGeneric.htm